### PR TITLE
Add v0.10 blog and images

### PR DESCRIPTION
**Pull request by pnbrown**
Add v10 blog and images

## What this PR does / why we need it

Adds blog for v0.10 release

## Details for the Release Notes (PLEASE PROVIDE)

```release-note
NONE
```

## Which issue(s) this PR fixes

Fixes: #

## Describe testing done for PR

Rendered site locally

### CODE REVIEW

File: docs/site/content/posts/05-tce-10-available.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+enhances usability on your local system, makes it easier to customize
+your platform, and simplifies IP address management across all nodes in
+a cluster. Today's release follows the recent [addition of build automation](https://tanzucommunityedition.io/posts/tanzu-community-edition-adds-kpack/)
+capabilities into Tanzu Community Edition as the team continues to
+deliver the power of Tanzu to you in a free, open source distribution.
+
+Here is a quick summary of the highlights of the new release:
+
+- The 0.10 release introduces a new 'unmanaged' cluster type that
+    significantly improves the efficiency of Tanzu Community Edition
```
Comment (stmcginnis) [inline docs/site/content/posts/05-tce-10-available.md:25]: Bullet item continuation lines are over indented (should align with the "T" of "The" in the first line. That said, it appears to render fine, so as long as nothing changes in the renderer, it should be fine.

**Review by stmcginnis — APPROVED** [review]
Looks fine to me. Just need to wait for official announcement time to merge.

File: docs/site/content/posts/05-tce-10-available.md (hunk 1 above)
Comment (pnbrown) [reply docs/site/content/posts/05-tce-10-available.md:25]: Right. There was a listing issue where there needed to be only 1 space after a bullet and the bullets all started out a that spacing. I saw it was rendering properly so I didn't touch it. I'm happy to make that correction though.

Comment (dvonthenen) [reply docs/site/content/posts/05-tce-10-available.md:25]: looks like all linting has passed resolving

**Pull request merged**